README.md

```diff
@@ -9,6 +9,7 @@ NUnit is a unit-testing framework for all .NET languages. Initially ported from
 ## Table of Contents ##
 
 - [Downloads](#downloads)
+- [Debugger source-stepping](#debugger-source-stepping)
```

Comment:
I like this, but disagree with listing it above `Documentation` - in terms of user-priority. How about moving it to between contributing and licence? (Would also be nice in that this is very relevant to people thinking of contributing.)